src/app/authentication/authentication.component.ts

```diff
@@ -84,6 +93,17 @@ export class AuthenticationComponent extends GraphExplorerComponent {
     }
   }
 
+  private isAppOutdated(current: string, previous: string): boolean {
+    const [ currMajor, currMinor, currPatch ] = current.split('.').map((value) => Number(value));
```

Comment:
You could simply compare the entire string without splitting on parts and get the same result.